environments/dev-env/CONTRIBUTING.md

```diff
@@ -0,0 +1,510 @@
+# KCP-Edge Contributor Development Environment
+
+## Required Packages:
+   - ko: https://ko.build/install/ 
+   - gcc
+   - docker 
+   - jq
+   - make
+   - go (version expected 1.19)
+   - kind
+   - kubectl  
+
+
+## Supported OS Platforms 
+  - Linux
+  - MacOS
+  - Windows WSL/Ubuntu
+
+
+Run the following script to install the required package (Linux or MacOS ):
+
+```bash
+./install_req.sh
+```
+
+Alternatively, run the following command for MacOS platform:
+
+```bash
+brew install ko gcc jq make go@1.19 kind kubectl
+```
+
+For Windows WSL/Ubuntu platform, follow the instructions [here](docs/wls_ubuntu_platform.md)
+
+
+## Quickstart
+
+### 1. Clone the kcp-edge repo:
+
+```shell
+git clone https://github.com/kcp-dev/edge-mc.git  kcp-edge
+```
+
+### 2. Build the kcp-edge binaries:
+
+```shell
+cd kcp-edge
+make build
+```
+
+The kcp-edge binaries is created inside the `/bin` directory. Add this directory to your $PATH:
+
+```shell
+export PATH="$PATH:$(pwd)/bin"
+```
+
+
+### 3. Create your own edge infrastructure (edge clusters) - Bring Your Own Cluster (BYOC)
```

Comment:
This header mixes plural and singular; let's not.
"BYOC" is our own internal jargon.  I do not think we need to highlight this piece of jargon.  It is the normal usage pattern.  We should just show users doing what we expect users to do.

Reply:
In our `demo`  environment using the `kcp-playground`, we are providing a `kind` for users with a sample workload deployed, the term BYOC clearly makes this distinction here.